src/main/java/com/github/complate/ComplateViewResolver.java

```diff
@@ -20,15 +20,19 @@
     public ComplateViewResolver(final ScriptingEngine scriptingEngine,
                                 final Resource bundle) {
         Assert.notNull(scriptingEngine, "ScriptingEngine must not be null");
-        Assert.notNull(bundle,
-            "bundle must not be null");
+        Assert.notNull(bundle, "bundle must not be null");
         this.scriptingEngine = scriptingEngine;
         this.bundle = bundle;
     }
 
+    public ComplateViewResolver(final Resource bundle) {
+        Assert.notNull(bundle, "bundle must not be null");
```

Comment:
I agree with @mvitz about always delegating to a primary constructor using `this`. I am also fine with changing the order of the parameters. Right now, we are still free to do these kinds of breaking changes.